packages/eslint-config/src/flat/configs/disables.ts

```diff
@@ -7,8 +7,11 @@ export default function disables(options: NuxtESLintConfigOptions): FlatConfig[]
   const nestedGlobPattern = `**/*.${GLOB_EXTS}`
 
   const fileRoutes = [
-    relative(dirs.src || '', `app.${GLOB_EXTS}`),
```

Comment:
Not related to this, but `dirs.src` is an absolute path, FWIW, and I don't think would do what we want here.

Also, wondering if we should distinguish rootDir from srcDir? Some files, like `nuxt.config` are always resolved from `rootDir` but others are resolved from `srcDir`

Reply:
I suppose it was a typo, and yeah I agree we should introduce rootDir as well!